src/Categories/Category/Construction/Graphs.agda

```diff
@@ -488,8 +488,8 @@ CatF = record
   open GraphMorphism≈
 
 -- Because of the Level changes in CatF, sizes must all be same:
```

Comment:
oops, forgot to remove the comment

Reply:
No worries, I'll take care of that later.